### ISSUE TITLE
Crewlink has stopped working since the update

### ISSUE THREAD

**Issue by FN-Florian** [ottomated/CrewLink#623]
# You MUST use this template or your issue will be deleted.

# Pre-Flight Checklist
Please use this checklist to avoid spamming:

1. [ X ] I am not asking a question => use the [Discord](https://discord.gg/9mwuVNA) if you are
2. [ X ] I am using the Steam version of Among Us
3. [ X ] I have tried to use an [alternative voice server](https://status.crewl.ink/)
4. [ X ] I have checked everyone in my lobby is on the same CrewLink server
5. [ X ] I have used `Ctrl+R` on the CrewLink app when I can't hear some people (this is a known issue)
6. [ X ] I have checked that the CrewLink server I'm using is up to date
7. [ X ] I have a screenshot of any errors
8. [ X ] I have checked that someone else has not reported this using the [search bar](https://github.com/ottomated/CrewLink/issues?q=is%3Aissue)

**Describe the bug**
Since the update to v2021.3.5s I only see the item "Menu" and no lobby in Crewlink.

**To Reproduce**
Steps to reproduce the behaviour:
1. Join a lobby in Among Us
2. Start Crewlink

**Expected behaviour**

**Screenshots**
![grafik](https://user-images.githubusercontent.com/17050701/110242153-0bb08e80-7f55-11eb-9037-a73e9ecda8dd.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: AmongUs: 2021.3.5s ; Crewlink: v2.0.1

**Additional context**

**Comment by KridyPlayz** [comment]
Same here

**Comment by reipowertools** [comment]
Same here

**Comment by NadineNightmare** [comment]
Dito

**Comment by Idenjant** [comment]
Yup, same problem

**Comment by royblokker** [comment]
Same here as well

**Comment by P3nguino** [comment]
Same here

**Comment by nextrie** [comment]
Same here

**Comment by ednese** [comment]
Same here >:(

**Comment by ednese** [comment]
Same here

**Comment by thomasjze** [comment]
😡

**Comment by AugZM** [comment]
same here, tried everything too

**Comment by P3nguino** [comment]
> My solution to this problem was to download [BetterCrewLink](https://github.com/OhMyGuus/BetterCrewLink/releases/latest) (a modified version of CrewLink) which supports the new version of Among Us, with that mine is no longer in the "MENU" infinitely even if you are in a lobby or in a match.

This was my fix too! Thank you!

**Comment by Ashrindy** [comment]
same

**Comment by darkesteart** [comment]
Yeah everyone should just forget about this being updated anytime soon, especially as Ottomated has made it clear that he works exclusively for content creators now at $10,000 a pop. Us lowly people aren't on his radar anymore 👍

**Comment by MatadorProBr** [comment]
> Yeah everyone should just forget about this being updated anytime soon, especially as Ottomated has made it clear that he works exclusively for content creators now at $10,000 a pop. Us lowly people aren't on his radar anymore 👍

He says he is doing CrewLink 3.0, and that it will be a huge update with a lot of new things and fixing this new problem, but it is taking a little time because Ottomated has a college and he is selling his mods for $ 10,000 to pay for his college, but I'm not defending him completely, because there are people who say that he will leave CrewLink 3.0 with closed code, that means that only a few people will be able to access CrewLink, and I think this is a absurd on his part, but anyway, one more reason for you to use BetterCrewLink that it already comes with several new things that you don’t have with the normal CrewLink like mobile support, listening to people on cameras, optimizations in the app, turning down the volume of a certain player, working in the new update of Among Us (2021.13.5) and many more things you can see the list [here](https://github.com/OhMyGuus/BetterCrewLink/wiki/Extra-features-fixes-in-this-fork)

**Comment by MatadorProBr** [comment]
Now CrewLink is dead, there is nothing else (I think) you can do but use BetterCrewLink to fix this problem, I enjoyed the good moments I used it.

R.I.P CrewLink
2020 - 2021
![image](https://user-images.githubusercontent.com/71022656/120080329-ba64f680-c08e-11eb-80a2-75ce8a0f93bc.png)
